src/Providers/ElasticApmServiceProvider.php

```diff
@@ -203,10 +203,18 @@ protected function listenForQueries()
 
             $stackTrace = $this->getStackTrace();
 
-
             // SQL type, e.g. SELECT, INSERT, DELETE, UPDATE, SET, ...
             $queryType = strtoupper(strtok(trim($query->sql), ' '));
 
+            // normalize the query
+            $statement = trim($query->sql);
+            // remove subsequent "?, " placeholders in IN () condidtions
+            // e.g. `tags`.`id` in (?, ?, ?, ?, ?, ..., ?, ?, ?))
+            $statement = preg_replace('#(\?, )+#', '?, ', $statement);
```

Comment:
Shouldn't this be
```suggestion
            $statement = preg_replace('#(\?, )+#', '', $statement);
```
?

Reply:
I want `in (?, ?, ?, ?, ?, ..., ?, ?, ?)` to be normalised to `in (?, ?)` to show that there were more values provided.

Reply:
I see, it's fine then 🙂